src/Text/Pandoc/Writers/Docx.hs

```diff
@@ -975,7 +975,11 @@ listItemToOpenXML opts numid bs = do
         _             -> False
   -- Prepend an empty string if the first entry is another
   -- list. Otherwise the outer bullet will disappear.
-  let bs' = if isListBlock (head bs) then (Plain [Str ""]:bs) else bs
+  let bs' = case bs of
+                 [] -> []
```

Comment:
ghc 9 now gives a "redundant pattern match" warning; it has gotten smart.
We could get rid of that by deleting 968.

Reply:
Sorry for requesting review before CI was complete. I have an older ghc version locally, and thought what could possibly go wrong? I've removed the line now. Thanks for the review!